core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2322,7 +2321,6 @@ class KafkaApis(val requestChannel: RequestChannel,
 
   def handleListPartitionReassignmentsRequest(request: RequestChannel.Request): Unit = {
     authorizeClusterDescribe(request)
-    val listPartitionReassignmentsRequest = request.body[ListPartitionReassignmentsRequest]
```

Comment:
Thanks for these, I'm the author which introduced them. There's not a real need to remove these nor `handleAlterPartitionReassignmentsRequest` since the KIP that implements them is 2/4th of the way implemented (as you can see we raise an unsupported version error below). There is a PR outstanding which implements the logic in these methods

Reply:
@stanislavkozlovski I see. Thanks for the context. I reverted this change.